### DISCUSSION TITLE
GSoC'23

### DISCUSSION THREAD

**Discussion by Samrj12** [gis-ops/valhalla-app#110] [General]
Hello everyone,

Just wanted to introduce myself here and meet other contributors who are also interested in GSoC. I am Rudraksh, a sophomore at the University of Toledo, studying Computer Science. I have experience working with Java, Javascript and the GoogleMaps API.

Excited to work with you all!

**Comment by Ruthvik2707** [comment]
Hello everyone can anyone explain how this gsoc supposed to work this repo popped in my git can any explain the process
